chpl/chpl-service/src/main/java/gov/healthit/chpl/job/SurveillanceUploadJob.java

```diff
@@ -181,17 +197,21 @@ public void run() {
             CertifiedProductDTO owningCp = null;
             try {
                 owningCp = cpManager.getById(surv.getCertifiedProduct().getId());
-                survValidator.validate(surv);
-                survManager.createPendingSurveillance(owningCp.getCertificationBodyId(), surv);
+                survValidator.validate(surv, false);
+
+                //TODO - This needs to be fixed
```

Comment:
Does this actually need more work?

Reply:
There is some sort of issue when trying to call @PostFilter.  The exception has something to do with retrieving ACL info.  I can't figure out why it's happening.  I spent an hour or so trying to figure out the problem and got no where.  I could spend more time trying to figure it out....  Thoughts?

Reply:
Addressed.